net-vpn/i2pd/i2pd-2.45.1.recipe

```diff
@@ -0,0 +1,74 @@
+SUMMARY="I2P: End-to-End encrypted and anonymous Internet"
+DESCRIPTION="i2pd (I2P Daemon) is a full-featured C++ implementation of I2P client.\
+I2P (Invisible Internet Protocol) is a universal anonymous network layer.\
+All communications over I2P are anonymous and end-to-end encrypted, participants \
+don't reveal their real IP addresses.\
+I2P client is a software used for building and using anonymous I2P networks. \ 
+Such networks are commonly used for anonymous peer-to-peer applications \
+(filesharing, cryptocurrencies) and anonymous client-server applications \ 
+(websites, instant messengers, chat-servers).\
+I2P allows people from all around the world to communicate and share information without restrictions.
+"
+HOMEPAGE="https://i2pd.website/"
+COPYRIGHT="i2pd developers"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/PurpleI2P/i2pd/archive/refs/tags/$portVersion.tar.gz"
+CHECKSUM_SHA256="a84b1e3cb5ac45f39af98e63291d34725ef67337e11acbe0e245acde7a5b2b72"
+PATCHES="$portName-$portVersion.patchset"
+SOURCE_DIR="$portName-$portVersion"
```

Comment:
$portName should not be used here, doesn't work for secondary architectures: 
`Error: i2pd_x86-2.45.1 doesn't exist in sources! Define SOURCE_DIR in recipe?`
Build error on buildmaster: https://build.haiku-os.org/buildmaster/master/x86_gcc2/logviewer.html?buildruns/5114/builds/81942.log